test/adapter/json_api/links_test.rb

```diff
@@ -7,18 +7,24 @@ class LinksTest < ActiveSupport::TestCase
         LinkAuthor = Class.new(::Model)
         class LinkAuthorSerializer < ActiveModel::Serializer
           link :self do
-            href "//example.com/link_author/#{object.id}"
+            href "http://example.com/link_author/#{object.id}"
```

Comment:
I just noticed there are several other areas that use http URLs. I'd advocate for changing them all in a single, separate PR.